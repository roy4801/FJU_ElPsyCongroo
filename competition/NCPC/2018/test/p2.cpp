#include <bits/stdc++.h>

using namespace std;

#define TESTC ""
#define PROBLEM "p2"

#define USE_CPPIO() ios_base::sync_with_stdio(0); cin.tie(0)

int main()
{
	#ifdef DBG
	freopen(PROBLEM TESTC ".in", "r", stdin);
	freopen(PROBLEM ".out", "w", stdout);
	#endif
	USE_CPPIO();

	int times;
	string s;

	cin >> times;
	
	while(times--)
	{
		cin >> s;

		cout << fixed << setprecision(6) << 1.f / stof(s) << '\n';		
	}


	return 0;
}