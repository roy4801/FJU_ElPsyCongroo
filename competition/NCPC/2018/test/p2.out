inf
1.000000
0.500000
0.333333
0.250000
0.200000
0.000000
nan
